app/views/aeon_pages/_info_modal.html.erb

```diff
@@ -0,0 +1,31 @@
+<div id="aeon-info-overlay" class="confirmation-overlay">
+  <div class="modal-body">
+    <h1 class="dialog-title" id='dialogTitle'><%= t('.header', library: Settings.libraries[i18n_library_title_key].label) %></h1>
+    <p><%= t('.reading_room_info') %></p>
+    <h2 class="dialog-how-to"><%= t('.how_to.header') %></h2>
+    <p><%= t('.how_to.body') %></p>
+    <p><%= t('.steps.header') %></p>
+    <ol class="dialog-steps">
+      <% if current_request.finding_aid? %>
+      <%= t('.steps.finding_aid_html') %>
+      <% elsif current_request.holdings.many? %>
+      <%= t('.steps.multi_vol_html') %>
+      <% else %>
+      <%= t('.steps.single_vol_html') %>
+      <% end %>
+    </ol>
+    <p class="dialog-more-info">
+      <%= t('.more_details_html', library: Settings.libraries[i18n_library_title_key].label, reading_room_url: Settings.libraries[i18n_library_title_key].reading_room_url) %>
```

Comment:
```suggestion
      <%= t('.more_details_html', library: Settings.libraries[current_request.origin].label, reading_room_url: Settings.libraries[current_request.origin].reading_room_url) %>
```

My read of https://github.com/sul-dlss/sul-requests/commit/f63f940d310c13db62372014d21e2be3770f91f4 suggests it was implemented specifically for I18n purposes, and might not be the library code?

I wonder if adding a `Request#library_metadata` access or something would be even better?